stdlib/types.pyi

```diff
@@ -142,28 +159,28 @@ class ModuleType:
     __spec__: Optional[ModuleSpec]
     def __init__(self, name: str, doc: Optional[str] = ...) -> None: ...
 
-class GeneratorType:
+class GeneratorType(Generator[_T_co, _T_contra, _V_co]):
```

Comment:
Shouldn't this also be generic?

Reply:
I didn't think so, but now I'm not so sure anymore.

Reply:
PEP 484 says:

> The Generic[T] base class is redundant in simple cases where you subclass some other generic class and specify type variables for its parameters:

So you are correct.

Reply:
PEP 484 says "The Generic[T] base class is redundant in simple cases where you subclass some other generic class and specify type variables for its parameters". https://www.python.org/dev/peps/pep-0484/#user-defined-generic-types